### ISSUE TITLE
Scheduled Task Revision

### ISSUE THREAD

**Issue by Marc05** [W4RH4WK/Debloat-Windows-10#27]
There are at least a few scheduled tasks which get disabled when running the disable-scheduled-tasks.ps1 script that are essential for general use. For example, "\Microsoft\Windows\SystemRestore\SR" which creates regular system restore points. Some others also make assumptions about the configuration of the system hardware, such as the ones that disable the maintenance tasks for SSHD's.

I propose that script should be generic by default, only disabling things such as telemetry collection, and have more specific configurations as optional.

**Comment by W4RH4WK** [comment]
Yeah this file part requires some more revision. I toggled most of the scheduled tasks to remain enabled.

**Comment by W4RH4WK** [comment]
I moved them to the utilities folder for now.

**Comment by W4RH4WK** [comment]
Maybe copy over these few:

<https://github.com/Nummer/Destroy-Windows-10-Spying/blob/master/DWS/MainDwsForm.cs#L712-L771>
